pkg/api/types.go

```diff
@@ -2890,6 +2890,10 @@ type ConfigMap struct {
 	// Data contains the configuration data.
 	// Each key must be a valid DNS_SUBDOMAIN with an optional leading dot.
 	Data map[string]string `json:"data,omitempty"`
+
+	// Data contains the binary data.
+	// Each key must be a valid DNS_SUBDOMAIN with an optional leading dot.
+	BinaryData map[string][]byte `json:"binaryData,omitempty"`
```

Comment:
Really this should be:

```
Data map[string]struct{
  String string
  Binary []byte
}
```

But we can't do that without breaking every client. :( This is why maps to raw types are bad in an API.